src/main/java/ch/njol/skript/variables/VariablesStorage.java

```diff
@@ -177,6 +201,9 @@ protected <T> T getValue(SectionNode sectionNode, String key, Class<T> type) {
 	 * @return whether the loading succeeded.
 	 */
 	public final boolean load(SectionNode sectionNode) {
+		String name = sectionNode.getKey();
+		databaseName = name;
```

Comment:
Is `name` used elsewhere? otherwise
```suggestion
		databaseName = sectionNode.getKey();
```